apps/www/registry/tui/ui/input.tsx

```diff
@@ -145,7 +145,12 @@ const Input = React.forwardRef<
   VariantProps<typeof inputVariants> &
   InputProps>(({ name, error, keyboardName,value,onChange,onInput,onKeyDown,onSubmit, alignIcon="left", iconStyle, icon, addOnBorder, textColor, labelAlign = "left", buttonLabelText, trailingAddOn, options, variant, placeholder, label, borderStyleForAddOn, alignDropdown, addOnText, hint, bottomBorder, disabled, border, note, asChild = false, className, ...props }, ref) => {
     const Comp = asChild ? Slot : "input"
-   
+    
+    const [inputValue, setInputValue] = useState('');
```

Comment:
@Parag27-cakewalk - Can we change the name to something which is valid for error and not for input value?